pkg/http-client/httpclient.go

```diff
@@ -13,15 +13,85 @@ limitations under the License.
 package httpclient
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"time"
+)
+
+const (
+	defaultTimeoutSeconds = 10
 )
 
 type Client interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
+//
+// creates a new instance of Client, given a path to addtional certificates
+// certFile may be empty string, which means no additional certs will be used
```

Comment:
I think this copy-n-paste needs updating :)

Reply:
indeed